Python/fitnesse/html_util.py

```diff
@@ -2,41 +2,41 @@
 
 from fitnesse.context import SuiteResponder, PageCrawlerImpl, PageData, PathParser, WikiPage, WikiPagePath
 
-
+class TestType:
+    SETUP: "SetUp"
+    TEARDOWN: "TearDown"
```

Comment:
Strictly speaking, it is not wrong to present the different types of tests using a class. However, the use of a dictionary or an enumeration is more lightweight and appropriate in this context.